README.md

```diff
@@ -1 +1,27 @@
-# software-process
\ No newline at end of file
+# software-process
+
+## Organization
+
+### Grading Requirements
+Alex, with his teaching experience, will make a grading scheme which will form the guideline to our self grading.
+From this scheme we can then elicit the requirements for a good grade and keep them in mind during our writing sessions.
+
+### Mechanism for frame of reference
+We're basing our process on a [scrum](http://www.scrumguides.org/scrum-guide.html)-like way of working.
+We're using only the core of scrum, which is the [scrum-board](https://www.mountaingoatsoftware.com/agile/scrum/task-boards) hosted on [trello](https://trello.com).
+
+### Work break-down
+We're breaking up the work by converting all sub-assignments in the blackboard document into tickets on our scrum-board, further dividing them up in smaller tasks as needed.
+
+### Quality and progress assurance
```

Comment:
Saying something about the verify step in Trello. Every work is being reviewed by at least one set of eyes.